### DISCUSSION TITLE
Trusted domain not accessible

### DISCUSSION THREAD

**Discussion by ThierryHFR** [nextcloud/all-in-one#4311] [Questions] [answered]
I used "NextCloud AIO borgbackup" to change my nextcloud machine installation
I was on an x86, I migrated to a raspberry pi 4.
The backup restore went well, but I can't access nextcloud through my domain.
Any ideas?

[nextcloud-aio.log](https://github.com/nextcloud/all-in-one/files/14478047/nextcloud-aio.log)

**Comment by ThierryHFR** [comment]
I think, despite your classification, that this is a bug.
Transferring Nextcloud-aio to a new machine is a pain. 
I tried to transfer the container, but it didn't work.
I tried to reinstall, but I run into the same problem as the https://github.com/nextcloud/all-in-one/discussions/4328 discussion.
No solution yet, I'm still on my old PC.

**Comment by szaimen** [comment, marked as answer]
Hi, can you post the output of `sudo docker logs nextcloud-aio-apache && sudo docker inspect nextcloud-aio-apache` here?

**Reply by ThierryHFR** [reply]
Hi @szaimen, 
Thank you for your feedback, here are the requested logs.
Strangely, the graphical interface tells me that everything is running...
[nextcloud-aio-apache.log](https://github.com/nextcloud/all-in-one/files/14508421/nextcloud-aio-apache.log)
![Capture d’écran du 2024-03-06 11-41-34](https://github.com/nextcloud/all-in-one/assets/10682892/1a9402e7-2ea0-4648-9c6b-bb36dd944615)

**Reply by szaimen** [reply]
Seems like your domain is not reachable. Have you set up your reverse proxy correctly to point at port 8001 of the server running aio?

**Reply by ThierryHFR** [reply]
Absolutely not, I didn't do anything special on the previous installation, I proceeded in the same way. 
Can you recommend a page of documentation?

**Reply by szaimen** [reply]
https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
